lib/ruby/lib/alces/stack/boot/run.rb

```diff
@@ -37,7 +37,10 @@ def initialize(options={})
           @finder = Alces::Stack::Templater::Finder.new("#{ENV['alces_BASE']}/etc/templates/boot/", options[:template])
           @group = options[:group]
           @dry_run_flag = options[:dry_run_flag]
+          @permanent_boot_flag = options[:permanent_boot_flag]
```

Comment:
```def template_parameters
      @template_parameters ||= {
        firstboot: true,
        permanentboot: permanent_boot_flag,
        kernelappendoptions: kernel_append.chomp
      }
    end
```
For this method, wouldn't the updated version of @template_parameters would be in the options scope and not in the Run scope?